osvr/RenderKit/RenderManagerBase.cpp

```diff
@@ -1525,16 +1510,12 @@ namespace renderkit {
                   (timing.timeUntilNextPresentRequired.microseconds / 1e3f);
               }
 
-              // Adjust the time at which the most-recent tracking info was
-              // set based on whether we're supposed to override it with "now".
-              // If not, find out how long ago it was.
+              // Find out how long ago this tracker info was found.
               float msSinceTrackerReport = 0;
-              if (!m_params.m_clientPredictionLocalTimeOverride) {
```

Comment:
Why is the optional local-time override being removed?  Do we know that future builds will all be VS2015?